components/style/properties/longhand/position.mako.rs

```diff
@@ -220,3 +220,15 @@ ${helpers.single_keyword("object-fit", "fill contain cover none scale-down",
                               spec="https://drafts.csswg.org/css-grid/#propdef-%s" % longhand,
                               products="gecko")}
 % endfor
+
+${helpers.predefined_type("grid-row-gap",
+                          "LengthOrPercentage",
```

Comment:
OK. so do i have to implement it using the template based on https://github.com/servo/servo/wiki/Property-hacking-guide ?